ci/test/pipeline.template.yml

```diff
@@ -137,7 +137,7 @@ steps:
         inputs:
           - "*"
         depends_on: []
-        timeout_in_minutes: 10
+        timeout_in_minutes: 20
```

Comment:
The step is called lint-fast 😀

Reply:
I want to see what is taking that long, I have a guess already